### PR TITLE
GD-255: Fix `runtest.cmd` to run `GdUnitCopyLog` in headless mode

### PR DESCRIPTION
**Pull request by LinuxUserGD**
# Why
The 'runtest.cmd' scripts still contains the '--no-window' argument where is changed with Godot4 to `--headless`

# What
Use `--headless` argument to execute `GdUnitCopyLog` in headless mode

### CODE REVIEW

File: addons/gdUnit4/runtest.cmd [hunk 1]
```diff
@@ -16,9 +16,9 @@ IF "%GODOT_TYPE%" == "mono" (
 	ECHO done %errorlevel%
 )
 
-%GODOT_BIN% -s -d .\addons\gdUnit4\bin\GdUnitCmdTool.gd %*
+%GODOT_BIN% --headless -s -d .\addons\gdUnit4\bin\GdUnitCmdTool.gd %*
 SET exit_code=%errorlevel%
-%GODOT_BIN% --no-window --quiet -s -d .\addons\gdUnit4\bin\GdUnitCopyLog.gd %*
+%GODOT_BIN% --headless --quiet -s -d .\addons\gdUnit4\bin\GdUnitCopyLog.gd %*
```
Comment (MikeSchulze) [inline addons/gdUnit4/runtest.cmd:21]: good catch, I overlooked this obsolete parameter.

File: addons/gdUnit4/runtest.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ if [ -z "$GODOT_BIN" ]; then
     exit 1
 fi
 
-$GODOT_BIN --path . -s -d ./addons/gdUnit4/bin/GdUnitCmdTool.gd $*
+$GODOT_BIN --headless --path . -s -d ./addons/gdUnit4/bin/GdUnitCmdTool.gd $*
```
Comment (MikeSchulze) [inline addons/gdUnit4/runtest.sh:9]: The `--headless` mode is currently forbidden to use. Godot has still a bug with `headless` and mouse event processing.

https://github.com/godotengine/godot/issues/73557

**Review by MikeSchulze — CHANGES_REQUESTED** [review]
running test with --headless mode is actually not supported, see my comments

File: addons/gdUnit4/runtest.cmd (hunk 1 above)
Comment (LinuxUserGD) [reply addons/gdUnit4/runtest.cmd:21]: I've removed the `--headless` parameter for the first command

Comment (MikeSchulze) [reply addons/gdUnit4/runtest.cmd:21]: can you please anwser the questions on the issue https://github.com/MikeSchulze/gdUnit4/issues/249

**Review by MikeSchulze — CHANGES_REQUESTED** [review]
@LinuxUserGD I have created a new bug issue to assign to this problem

**Review by MikeSchulze — APPROVED** [review]

**Pull request merged**